test/replays/132/replays.test.ts

```diff
@@ -141,3 +141,11 @@ it("parses 1.32.8 replay with observer on defeat setting", async () => {
   );
   expect(test.settings.observerMode).toBe("ON_DEFEAT");
 });
+
+it("should parse hotkeys correctly", async () => {
+  const test = await Parser.parse(path.resolve(__dirname, "reforged1.w3g"));
+  expect(test.players[0].hotkeys[1]).toEqual({ assigned: 1, used: 29 });
```

Comment:
Please add the new property to the jsonschema aswell.

Reply:
Do you do that manually?

Reply:
Yes, the JSONSchema is a manually edited document and the output is asserted to match it in a unit test.